### DISCUSSION TITLE
When deleting a repository, the timeline does not change

### DISCUSSION THREAD

**Discussion by criskell** [community/community#84769] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello, I created a repository with a commit with the date specified for my birth, for fun purposes.

But when I deleted the repository, my timeline remained unchanged.

When I go to the last year of the timeline, there are no contributions there, no commits.

**Comment by georgetoloraia** [comment]
Testing answer:
- Answer 1
- Answer 2
- [ ] one
- [x] two

**Comment by ibra-kdbra** [comment]
**Good evening**, when you delete a repository on GitHub, the contributions from that repository are removed from your contribution graph. It might take some time for the changes to reflect on your contribution graph.

**Comment by DTS-11** [comment]
Deleting a repository on GitHub doesn't remove commits from your contributions timeline. If you want to erase specific commits, rewrite the Git history locally and force-push changes to GitHub. Alternatively, use `git commit --amend` for individual commits. Keep in mind that force-pushing can be powerful, so use it cautiously. Changes may take time to reflect on your GitHub timeline. To avoid impacting your main profile, consider creating private repositories or forking before making playful commits.
